scripts/_common.sh

```diff
@@ -41,8 +41,10 @@ init_virtualenv () {
 configure_nginx () {
     local domain=$1
     local path=$2
+    local app_home=$3
```

Comment:
How this new argument will behave with upgrade script, where `configure_nginx` is also called.